### PR TITLE
Bump version and changelog dates

### PR DESCRIPTION
**Pull request by marcoag**
Signed-off-by: Marco A. Gutierrez <marco@openrobotics.org>

### CODE REVIEW

File: rmf_fleet_msgs/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Changelog for package rmf_fleet_msgs
 
-1.3.0 (2021-XX-XX)
+1.3.0 (2021-05-27)
```
Comment (Yadunund) [inline rmf_fleet_msgs/CHANGELOG.md:3]: The release date for packages in `rmf_ros2` was updated to `2021-06-07`. Should that be the case for the packages here too?

**Review by Yadunund — COMMENTED** [review]
Looks good. But I have a small clarification on the release date in the Changelogs.

File: rmf_fleet_msgs/CHANGELOG.md (hunk 1 above)
Comment (marcoag) [reply rmf_fleet_msgs/CHANGELOG.md:3]: As packages have different versions and were released at different times I would see nothing wrong with having this date different between them. This way we also make sure the package includes all the changes up to the release date (`2021-05-27` in this case) whereas if we use `2021-06-07` changes made between `2021-05-27` and `2021-06-07`might not be included in the released package which can create some confusion.

**Review by Yadunund — APPROVED** [review]
Thanks for the clarification

**Pull request merged**